bin/mev/src/cmd/relay.rs

```diff
@@ -31,7 +30,7 @@ impl Command {
         let config = Config::from_toml_file(config_file)?;
 
         let network = config.network;
-        info!("configured for `{network}`");
+        tracing::info!("configured for `{network}`");
```

Comment:
generally prefer to import things so there is just a single term at the use site